### PR TITLE
[New] Add responsive w/ git auto-update

### PR DESCRIPTION
**Pull request by extend1994**
Related pull request: #11873

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/amcharts/responsive
 * Official website (optional, not the repository): https://www.amcharts.com/
 * License and its reference: [Apache-2.0](https://github.com/amcharts/responsive/blob/master/license.txt)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 7
   - Count of stars: 13
   - Count of forks: 13

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
1. Add this library though it has insufficient popularity because it is the plugin of other hosted libraries such as amcharts.
2. Thought there is no tag of the manual adding versions, they are used, for example, as the plugin of amcharts v3.13.0 ~ 3.14.0 (plugins appeared since v3.13.0). Take two reference for your reivew: https://github.com/amcharts/amstock3/blob/3.14.0/amcharts/plugins/responsive/responsive.js & https://github.com/amcharts/amstock3/blob/3.13.0/amcharts/plugins/responsive/responsive.js. This case can be also applied to the libraries amstockchart and ammap.

**Comment by sufuf3** [comment]
Could we add v0.9? https://github.com/amcharts/responsive/tree/6247844b182b4fad875f72aff91e3a41a1fe1482
It's initial release.

**Comment by extend1994** [comment]
@sufuf3 I had seen that version commit but I didn't add it because I think we still need to add library mainly depends on release/tag. The reason I manually add v0.9.1/0.9.2/1.0 is that they are used by other amchart libraries.
@PeterDaveHello This library has releases v1.0.1~v1.0.5. pacakge.json was created since v1.0.2. But readme.md and `responsive.js`also records version. Like what you can see in https://github.com/amcharts/responsive/blob/master/readme.md#amcharts-responsive and https://github.com/amcharts/responsive/blob/master/responsive.js#L6. I manually add v0.9.1/0.9.2/1.0 here because they are used by other amchart libraries. I am not sure can I manually add all old versions by tracing its version commit logs of readme.md and responsive.js?

**Comment by sufuf3** [comment]
I think we can just use `add lib process` to do this, as usual.
What do you think?

**Comment by extend1994** [comment]
@sufuf3 I just asked Peter do we need to add the versions we find in commit logs. He said we can ignore them if they were not requested specially. So I think it's fine to not add v0.9.

**Comment by PeterDaveHello** [comment]
Yes if there are not so many and the missing ones are not directly requested, but it really depends, requests from internal also counts, so @sufuf3 can call if we should add it, discussions would be welcome.

File: ajax/libs/responsive/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "responsive",
+  "filename": "responsive.min.js",
+  "version": "1.0.5",
+  "description": "amCharts V3 Responsive plugin",
```
Comment (maruilian11) [inline ajax/libs/responsive/package.json]: What's your ref of `description`?
I think we can use https://github.com/amcharts/responsive/blob/master/responsive.js#L3 to update package.json.

Comment (extend1994) [reply ajax/libs/responsive/package.json]: It's from homepage, but the one you mentioned seems better. I'll update it in a minute. Thanks for suggestion.

**Comment by extend1994** [comment]
@maruilian11 description had been updated, please help review it, thanks.

**Pull request merged**